OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -156,6 +156,7 @@
     <Compile Include="Primitives\MergedStream.cs" />
     <Compile Include="Primitives\SpatiallyPartitioned.cs" />
     <Compile Include="Primitives\ConcurrentCache.cs" />
+    <Compile Include="SelectableExts.cs" />
```

Comment:
I personally don't even see the gain of extracting it to a file on it's own over just adding it to the end of the Selectable's file.